chaos_genius/views/anomaly_data_view.py

```diff
@@ -425,6 +357,46 @@ def kpi_anomaly_retraining(kpi_id):
         return jsonify({"msg": f"retraining failed for KPI: {kpi_id}, KPI id is None"})
 
 
+def _get_dimensions_values(kpi_id):
+    """Creates a dictionaryy of KPI dimension and their values.
+
+    :param kpi_id: ID of the KPI
+    :type row: integer
```

Comment:
There's no param named `row` here
```suggestion
    :type kpi_id: integer
```

Reply:
Also, `integer` isn't a python type. `int` would make it clearer.

Reply:
Fixed.